zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/UserControllerTest.java

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import io.camunda.identity.usermanagement.CamundaUser;
+import io.camunda.identity.usermanagement.CamundaUserWithPassword;
+import io.camunda.identity.usermanagement.service.UserService;
+import io.camunda.zeebe.gateway.protocol.rest.CamundaUserDto;
+import io.camunda.zeebe.gateway.protocol.rest.CamundaUserWithPasswordDto;
+import io.camunda.zeebe.gateway.protocol.rest.SearchRequestDto;
+import io.camunda.zeebe.gateway.protocol.rest.UserSearchResponseDto;
+import io.camunda.zeebe.gateway.rest.GlobalControllerExceptionHandler;
+import io.camunda.zeebe.gateway.rest.controller.UserControllerTest.TestUserControllerApplication;
+import io.camunda.zeebe.gateway.rest.controller.usermanagement.UserController;
+import java.net.URI;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ProblemDetail;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+@SpringBootTest(
+    classes = {
+      TestUserControllerApplication.class,
+      UserController.class,
+      GlobalControllerExceptionHandler.class
+    },
+    webEnvironment = WebEnvironment.RANDOM_PORT)
+public class UserControllerTest {
+
+  @MockBean private JobController jobController;
+  @MockBean private ProcessInstanceController processInstanceController;
+  @MockBean private TopologyController topologyController;
+  @MockBean private UserTaskController userTaskController;
+  @MockBean private HttpSecurity httpSecurity;
+  @MockBean private UserService userService;
+
+  @Autowired private WebTestClient webClient;
+
+  @Test
+  void getUserByIdWorks() {
```

Comment:
🔧 For consistency and style, can we use wording like `shouldReturnExistingUserById` here and in other tests? This directly conveys what the test wants to achieve and what it expects.

Reply:
got it, thanks ... will be there with next push :)